CHANGELOG.md

```diff
@@ -205,6 +205,8 @@ generated user module:
 * Update documentation to mention `{:inner, selector}` and `{:closest, selector}` as [valid options for `to`](https://hexdocs.pm/phoenix_live_view/1.0.3/Phoenix.LiveView.JS.html#module-dom-selectors) in JS commands ([#3638](https://github.com/phoenixframework/phoenix_live_view/pull/3638))
 
 ## 1.0.2 (2025-01-09)
+### Breaking changes
```

Comment:
I'm not sure if I would classify it as a breaking change, as it does not change the behavior of the supported case (no duplicates). I'd say keep it as an enhancement, but maybe add something like.

> Note: this changes the behavior of stream inserts if your application relied on (unsupported) duplicate IDs

or something similar. cc @josevalim

Reply:
In my opinion it is a bug fix, because it would lead to broken behaviour on the client.